db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/deployment.rb

```diff
@@ -1,32 +1,18 @@
-def analyze_ansible_output(output)
-  results = output.rpartition('PLAY RECAP ********************************************************************').last
-  results = results.split("\r\n")
-  results.shift
-  passed = true
-  results.each do |node_result|
-    unless node_result.include?("unreachable=0") && node_result.include?("failed=0")
-      passed = false
-      $evm.root['ae_result'] = "error"
-      $evm.root['automation_task'].message = "deployment failed"
-      next
-    end
-    break unless passed
-  end
-
-  if passed
-    $evm.root['automation_task'].message = "successful deployment"
+$evm.log(:info, "********************** #{$evm.root['ae_state']} ******************************")
+begin
+  $evm.root['container_deployment'] = $evm.vmdb(:container_deployment).find(
+    $evm.root['automation_task'].automation_request.options[:attrs][:deployment_id]) unless $evm.root['container_deployment']
+  result = $evm.root['container_deployment'].run_playbook_command($evm.root['deployment_master'], $evm.get_state_var(:ssh_user), "atomic-openshift-installer -u -c /usr/share/ansible/openshift-ansible/inventory.yaml install 1> /tmp/ansible.log 2> /tmp/openshift-ansible.log.2 < /dev/null")
+  if result[:finished]
+    $evm.root['ae_result'] = $evm.root['container_deployment'].analyze_ansible_output(result[:stdout]) ? "ok" : "error"
```

Comment:
could you give ae_reason here?
is no exit MIQ_ERROR here correct?  (deliberate crash vs failure distinction)?